### DISCUSSION TITLE
【要望】キーボード操作の実装

### DISCUSSION THREAD

**Discussion by akrha** [increments/qiita-discussions#584] [要望・アイデア・不具合]
タイトルのとおりです。

いいね、ストック、コメント、新規記事作成などの主要な操作が、XやGitHub、Gmailなどのウェブアプリのようにキーボードから操作できるとキーボードで操作が完結できて便利です。
`j` `k` で項目の上下移動、 `l` でいいね、 `/` で検索にフォーカス、 `?` でキーボード操作一覧の表示といったイメージです。

**Comment by tanaka-asami415** [comment]
@akrha 
リクエストありがとうございます！
今後の改善の参考にさせていただきます✨
